src/Module/Profile/Photos.php

```diff
@@ -97,8 +116,16 @@ protected function post(array $request = [])
 			$str_contact_allow .= $this->aclFormatter->toString(Contact::getPublicIdByUserId($this->owner['uid']));
 		}
 
+		$hook_data = [
```

Comment:
standing for all other occurrences:

you move the `$request` from directly setting it as a parameter and instead move it to an array and passing it.
Is this compatible?